modules/kubernetes-addons/secrets-store-csi-driver/locals.tf

```diff
@@ -0,0 +1,30 @@
+locals {
+  name = "secrets-store-csi-driver"
+
+  default_helm_config = {
+    name        = local.name
+    chart       = local.name
+    repository  = "https://kubernetes-sigs.github.io/secrets-store-csi-driver/charts"
+    version     = "1.1.2"
+    namespace   = local.name
+    description = "A Helm chart to install the Secrets Store CSI Driver "
+    values      = []
+    timeout     = "1200"
+  }
+
+  helm_config = merge(
+    local.default_helm_config,
+    var.helm_config
+  )
+
+  irsa_config = {
+    create_kubernetes_namespace       = true
+    kubernetes_namespace              = local.name
+    create_kubernetes_service_account = false
+    kubernetes_service_account        = local.name
+  }
```

Comment:
Don't use irsa module if we are not creating an IAM role with service account. 
Simply create a Namespace object for this